packages/ruleset-bundler/src/plugins/commonjs.ts

```diff
@@ -0,0 +1 @@
+export { default as commonjs } from '@rollup/plugin-commonjs';
```

Comment:
How is consumer expected to import this - deep import through `/dist` or something? (or is not meant for general consumption)

Reply:
we have package exports, so can can import this as follows
```ts
import { commonjs } from '@stoplight/spectral-ruleset-bundler/plugins/commonjs';
```